packages/cacti-copm-core/openapitools.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "./node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "spaces": 2,
+  "generator-cli": {
+    "version": "7.7.0"
```

Comment:
@jenniferlianne We currently use 6.6.0 - deviating from that version will cause race conditions in the codegen script which only pre-fetches the jar for v6.6.0 (search for "warmup" in the root package.json to see what I mean).

Similar to my other comment about the dependency declaration syntax, here it is the same idea: We can (and I would love to) upgrade to the newer versions of the Open API generator because the newer templates have fixes and additional features (usually) but if we do it it's safer to do it in a single PR dedicated to it which then upgrades the whole project, otherwise there's additional breakages due to the inconsistencies.

Reply:
Addressed.